exercises/supervised-regression/ex_rnw/ex_polynomial.Rnw

```diff
@@ -0,0 +1,36 @@
+Assume the following (noisy) data-generating process: $y = 0.5 + \sin(2\pi x) + 
+\epsilon$ with $\epsilon ~ \sim \mathcal{N}(0, 0.1)$.
+
+<<echo=FALSE, message=FALSE, fig.align="left", fig.height=2.5, fig.width=4>>=
+
+n <- 50L
+x <- seq(0L, 1L, length.out = n)
+set.seed(123L)
+y <- 0.5 + 0.4 * sin(2 * pi * x) + rnorm(n, sd = 0.1)
+
+ggplot2::ggplot(data.frame(x = x, y = y), ggplot2::aes(x = x, y = y)) +
+  ggplot2::geom_point() + 
+  ggplot2::theme_bw()
+
+@
+
+\begin{enumerate}[a)]
+  \item We decide to model the data with a cubic polynomial (including intercept 
+  term). State the corresponding hypothesis space.
+  \item Demonstrate that this hypothesis space is simply a parameterized family 
+  of curves by plotting in \texttt{R} curves for 3 different models belonging to 
+  the considered model class.
+  \item State the empirical risk w.r.t. $\thetab$ for a member of our hypothesis 
```

Comment:
Das verstehe ich nicht ganz, also das "for a member": Dafür muss ich dann konkrete Werte für theta einsetzen? Wenn ja, verstehe ich das "wrt theta" nicht.